### PR TITLE
Fix and update E2E tests

### PR DESCRIPTION
**Pull request by rikukissa**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -5,5 +5,9 @@
   "viewportWidth": 1024,
   "viewportHeight": 768,
   "defaultCommandTimeout": 60000,
-  "videoCompression": 0
+  "videoCompression": 0,
+  "retries": {
+    "runMode": 2,
```
Comment (rikukissa) [inline cypress.json:10]: This makes Cypress retry failing tests 2 times

File: cypress/integration/reports.spec.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ context('Reports Integration Test', () => {
     indexedDB.deleteDatabase('OpenCRVS')
   })
 
-  it('Tests for performance reports birth using minimum input', () => {
+  it.skip('Tests for performance reports birth using minimum input', () => {
```
Comment (rikukissa) [inline cypress/integration/reports.spec.ts:19]: I skipped these tests as I think they might not be valid anymore. @azantor64 can you confirm?

File: cypress/integration/performance.spec.ts [hunk 3]
```diff
@@ -16,30 +16,24 @@ context('Performance view', () => {
...
   it('Tests from declaration to certification using minimum input', () => {
-    
     cy.declareDeclarationWithMinimumInput('Mr', 'Performance')
-  
   })
 
- 
-  it('Login as registrar to register & Downloads CSV data to observe Performance',() => {
-    
+  it.skip('Login as registrar to register & Downloads CSV data to observe Performance', () => {
```
Comment (rikukissa) [inline cypress/integration/performance.spec.ts:22]: This test needs to be updated to match current spec

File: cypress/integration/reports.spec.ts (hunk 2 above)
Comment (azaworld) [reply cypress/integration/reports.spec.ts:19]: Yeah. It should be skipped for now.

**Pull request merged**